client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -48,7 +52,7 @@ export class HubConnection {
         var descriptor = JSON.parse(data);
         if (descriptor.Method === undefined) {
             let invocationResult: InvocationResultDescriptor = descriptor;
-            let callback = this.callbacks[invocationResult.Id];
+            let callback = this.callbacks.get(invocationResult.Id);
```

Comment:
HubConnection [uses Map](https://github.com/aspnet/SignalR/blob/dev/client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts#L21-L22) not {}.